scripts/make-env

```diff
@@ -61,6 +61,9 @@ export OPENBALENA_VPN_SERVER_DH=$(b64file "$VPN_DH")
 export OPENBALENA_VPN_SERVICE_API_KEY=$(randstr 32)
 export OPENBALENA_API_VPN_SERVICE_API_KEY=$(randstr 32)
 export OPENBALENA_REGISTRY_SECRET_KEY=$(randstr 32)
+export OPENBALENA_S3_BUCKET=openbalena/registry
```

Comment:
This is not a valid bucket name, AFAIK. How about using `registry-data` to match balenaCloud?

The bucket also needs to preexist when the registry starts up and I don't see when/where it is created.

Reply:
Didn't know this; it seemed to work OK though. Weird. I will see what I can do.

Reply:
Paths don't really 'exist' in S3, it's more of a denoted filename which allows filtering to occur based on prefix, I think @dfunckt is right in saying it's not a good bucket name. There shouldn't be any delimiters.

Reply:
using `registry-data` now 👍